docs/submission/validating-molecular-data.md

```diff
@@ -266,37 +267,37 @@ Sequencing data of both `BAM` and `FASTQ` type files are accepted, with the exce
-|  **library_name**               | read_groups |  ![Required](/assets/submission/dictionary-required.svg)  |  Name of a sequencing library made from a molecular sample or a sample pool (multiplex sequencing).                                                                                                                                                                                                                                                                          |                      |  
-|  **fileName**       | files |  ![Required](/assets/submission/dictionary-required.svg)  |  Name of the file.                                                                            |                      |  
-|  **fileSize**       | files |  ![Required](/assets/submission/dictionary-required.svg)  |  Size of the file, in bytes.                                                                  |                      |  
-|  **fileMd5sum**     | files |  ![Required](/assets/submission/dictionary-required.svg)  |  Compute the md5sum of the file. This must match what is computed when the file is uploaded.  |  |  |
-|  **fileType**       | files |  ![Required](/assets/submission/dictionary-required.svg)  |  Data format of sequencing files.                                                             |  BAM, FASTQ          |  
-|  **fileAccess**     | files |  ![Required](/assets/submission/dictionary-required.svg)  |  The permission level of a file for public access                                                              |  open, controlled|
-|  **dataType**      | files |  ![Required](/assets/submission/dictionary-required.svg)  |  Descriptor of the type of file being submitted.                                              |  Submitted Reads     | 
+|  **platform_model** | experiment |                                                           |  The model number of the sequencing machine used in data generation.                                     |   Any string value or `null`.                                 |  
+|**experimental_strategy**| experiment |  ![Required](/assets/submission/dictionary-required.svg)  |  The primary experimental method. For sequencing data it refers to how the sequencing library was made.  |  WGS, WXS, RNA-Seq, Bisulfite-Seq  |  
+|  **sequencing_date**  | experiment |                                                           |  Date sequencing was performed.      |datetime format, for example: 2019-06-16 or  2019-06-16T20:20:39+00:00  |  
+|  **read_group_count**  |  |  ![Required](/assets/submission/dictionary-required.svg)  |  The number of read groups in the molecular files being submitted.  | A minimum of 1 is required.  |  
```

Comment:
This is missing the section name

Reply:
it isnt under a section; its just at the base level.